src/components/notifications/Notifications.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { CSSTransitionGroup } from 'react-transition-group';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import Notification from './Notification';
+
+const propTypes = {
+  timeToLiveInSeconds: PropTypes.number
+};
+
+const defaultProps = {
+  timeToLiveInSeconds: 5
+};
+
+const Container = styled(CSSTransitionGroup)`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: absolute;
```

Comment:
```suggestion
  position: fixed;
```

This way it stays down when you scroll the page